backend/src/main/java/com/bikes/backend/service/BikeService.java

```diff
@@ -27,19 +25,19 @@ public Bike getBikeById(String id) {
 	}
 
 	public Bike addBike(BikeDTO incomingBike) {
-		Bike bikeToAdd = new Bike(idService.generateId(), incomingBike.title());
+		final String DEFAULT_BIKE_IMAGE = "https://res.cloudinary.com/diikwvy8u/image/upload/v1678453345/8-WnQm41fywTkFZPX_nywzmq.png";
+		Bike bikeToAdd = new Bike(idService.generateId(), incomingBike.title(), DEFAULT_BIKE_IMAGE);
```

Comment:
Why can I not upload an image while adding?

Reply:
Because you will not get a chance to see the dedicated default bike image in that case. ;-)